### PR TITLE
Fix broken links for Korean tutorial pages

### PR DESCRIPTION
**Pull request by gochist**
Fix broken links for images and pages in the [ko/tutorials/kubernetes-basics](https://kubernetes.io/ko/docs/tutorials/kubernetes-basics/)

### CODE REVIEW

File: content/ko/docs/tutorials/kubernetes-basics/_index.html [hunk 1]
```diff
@@ -45,25 +45,25 @@ <h2>쿠버네티스 기초 모듈</h2>
           <div class="row">
             <div class="col-md-4">
               <div class="thumbnail">
-                <a href="/docs/tutorials/kubernetes-basics/cluster-intro/"><img src="./public/images/module_01.svg?v=1469803628347" alt=""></a>
+                <a href="./create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
```
Comment (zacharysarah) [inline content/ko/docs/tutorials/kubernetes-basics/_index.html]: @gochist 👋 I tried the full path of `/ko/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/` instead, and immediately saw why you updated to more relative paths: 
```
http://localhost:1313/ko/docs/tutorials/kubernetes-basics/ko/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/
```

So thanks for these fixes. 👍

**Comment by zacharysarah** [comment]
@gochist Would these make good upstream contributions?

**Comment by gochist** [comment]
@zacharysarah Sure. I would happily do it.

**Pull request merged**